package.json

```diff
@@ -22,6 +23,7 @@
     "object-describer": "https://github.com/kubernetes-ui/object-describer.git#v1.0.4",
     "patternfly": "3.35.1",
     "patternfly-bootstrap-combobox": "1.1.7",
+    "patternfly-react": "1.16.5",
```

Comment:
I bumbed it a bit up to 1.19.1, but we cannot go higher that that. Because `patternfly-react` requires React 16 which is not supported by `react-lite`.

From react-lite GH: `Note: react-lite dose not support React v16.x now.`

Reply:
We should make an effort to move from `react-lite` to `react`